.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,16 @@
+---
+name: "\U0001F914 Question"
+about: Usage question that isn't answered in docs or discussion
+
+---
+
+## Question
+
+<!-- Before asking a question, please check:
+- TiKV Issue List: https://github.com/pingcap/tikv/issues?utf8=%E2%9C%93&q=is%3Aissue
+- TiKV Readme: https://github.com/pingcap/tikv
+- TiKV Doc: https://github.com/pingcap/tikv/wiki/TiKV-Documentation
+- TiKV Control Doc: https://pingcap.github.io/docs/tools/tikv-control/
+- TiDB & TiKV Stack Doc: https://pingcap.github.io/docs/
+- Google: https://www.google.com/
```

Comment:
This comes off kind of snarky. Maybe suggest they Google/StackOverflow the error?